java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -0,0 +1,60 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.geospatial.GeoAddOptions;
+import glide.api.models.commands.geospatial.GeospatialData;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Geospatial Indices Commands" group for standalone and
+ * cluster clients.
```

Comment:
Maybe mention something about Geospatial data is saved as a Sorted Set?  GEODEL is not needed because ZDEL already exists.

Reply:
Copied documentation from AWS don't think we need it.